src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyNonCacheableRequest.java

```diff
@@ -34,44 +32,57 @@
 import org.reaktivity.nukleus.http_cache.internal.types.stream.ResetFW;
 import org.reaktivity.nukleus.http_cache.internal.types.stream.WindowFW;
 
-public final class HttpCacheProxyNonCacheableRequest extends HttpCacheProxyRequest
+final class HttpCacheProxyNonCacheableRequest
 {
-
-    private Request request;
+    private final HttpCacheProxyFactory streamFactory;
+    private final long acceptRouteId;
+    private final long acceptStreamId;
+    private final long acceptReplyId;
+    private final MessageConsumer acceptReply;
+
+    private MessageConsumer connectInitial;
+    private MessageConsumer connectReply;
+    private long connectRouteId;
+    private long connectReplyId;
+    private long connectInitialId;
```

Comment:
Check which fields should also be `final`.

Reply:
Done.